src/utils/Utils.ts

```diff
@@ -1364,6 +1365,32 @@ export default class Utils {
     };
   }
 
+  public static buildPerformanceRecord(params: {
+    tenantID: string; durationMs: number; sizeKb?: number;
+    source?: string; module: string; method: string; action: ServerAction|string;
+    httpUrl?: string; httpMethod?: string; httpCode?: number;
+  }): PerformanceRecord {
+    return {
+      tenantID: params.tenantID,
+      timestamp: new Date(),
+      durationMs: params.durationMs,
+      sizeKb: params.sizeKb,
+      host: Utils.getHostname(),
+      process: cluster.isWorker ? 'worker ' + cluster.worker.id.toString() : 'master',
+      memoryTotalGb: Utils.truncTo(os.totalmem() / 1000000000, 3),
+      memoryFreeGb: Utils.truncTo(os.freemem() / 1000000000, 3),
```

Comment:
I missed that: ideally only the raw value are stored in performance record to allow fine grained statistic computation later on: mean, median, holtz-winter smoothing for forecasting, etc. 
If any rounding or truncation is need, it's done later on after reading the performance record.

Reply:
Okay. Just removed truncating.